packages/react/src/sprinkles/sprinkles.css.ts

```diff
@@ -55,235 +55,219 @@ const modifiers = {
   ),
 };
 
-export const props = [
-  defineProperties({
-    "@layer": layers.axiom,
-    conditions,
-    properties: {
-      alignItems: ["center", "end", "normal", "start", "stretch"] as const,
-      colSpan: {
-        "1": { gridColumn: "span 1 / span 1" },
-        "2": { gridColumn: "span 2 / span 2" },
+export const props = defineProperties({
+  "@layer": layers.axiom,
+  conditions,
+  modifiers,
```

Comment:
The type declaration output has ballooned in size because of combining everything - will need to investigate how to reduce that